docs/usage.md

```diff
@@ -54,7 +54,20 @@ Please note the following requirements:
 - Within one samplesheet either only single-end or only paired-end reads can be specified
 - If single-end reads are specified, the command line parameter `--single_end` must be specified as well
 
-Again, by default, the group information is only used to compute co-abundances for the binning step, but not for group-wise co-assembly (see the parameter docs for [`--coassemble_group`](https://nf-co.re/mag/parameters#coassemble_group) and [`--binning_map_mode`](https://nf-co.re/mag/parameters#binning_map_mode) for more information about how this group information can be used).
+### The `group` parameter
+
+In MAG reconstruction, there are different strategies on how to both _assemble_ and _bin_ the samples.
+These are usually referred to as "individual" or "single" assembly and binning, versus "co-" or "pooled" assembly and binning.
+In the former, samples are processed individually.
+In the latter, samples are processed together.
+A common strategy is to assemble each sample individually (single assembly) and then pool all of the contigs together and map the reads against them (co-binning).
+This is usually chosen because since assembly is a computationally intensive process, it is very costly to assemble all samples together, even if it were to lead to better contigs.
```

Comment:
```suggestion
This is usually chosen because since assembly is a computationally intensive process, it is very costly to assemble all samples together.
```

I think we want to be careful about saying co-assembly is better - it's context and sample-dependent and can lead to chimeric sequences